src/includes/performance/distributed-tracing/dotnet.mdx

```diff
@@ -0,0 +1,45 @@
+## Distributed Tracing
+
+Traces can bridge across multiple software services. Each span in a trace can be represented as a `sentry-trace` header, containing the trace id, span id, and sampling details. This `sentry-trace` header can be passed to downstream services so that they can create spans that are a continuation of the trace started in the originating service.
+
+To obtain a trace header from the span, use `ISpan.GetTraceHeader()` method, then pass it to the downstream service. If the communication happens over HTTP, it is recommended that you set the value to the `sentry-trace` HTTP header.
+
+```csharp
+var transaction = SentrySdk.StartTransaction("name", "op");
+var sentryTraceHeader = transaction.GetTraceHeader();
+
+using var request = new HttpRequestMessage(HttpMethod.Get, "https://example.com");
+request.Headers.Add("sentry-trace", sentryTraceHeader.ToString());
+```
+
+<Note>
+
+Sentry also provides `SentryHttpMessageHandler`, which can be used to automatically attach trace headers to outgoing HTTP requests. Learn more about it in <PlatformLink to="/performance/included-instrumentation/">this article</PlatformLink>.
```

Comment:
iirc `PlatformLink` is used for something else, not really just linking the user to a different page, correct me if I am wrong @PeloWriter

Reply:
I thought the approach from @Tyrrrz was correct. Wouldn't `platforms/dotnet/performance/included-instrumentation/` navigate the user out of one of the guides?

If the user is browsing this content through `/platforms/dotnet/guides/aspnetcore/performance/...` wouldn't we take them out of it?

Reply:
mm idk about it, if its the case, I'm pretty sure that every doc using Guides should be using `PlatformLink` instead and it's not the case.

Reply:
Couple of things:
- the link is missing the preceding `/` (my fault, as I didn't c/p properly) which is why the linkchecker is failing
- we use the PlatformLink when the specific platform isn't already selected

Reply:
fixed the links

Reply:
@PeloWriter what @bruno-garcia said makes sense, if you're in a Guide but using normal links instead of PlatformLink, once you click it, you move out of the Guide, is that the desired behavior? I'd think so

Reply:
We have been using this format - [click this](page to to which you want users to go). When we need the platform context to be derived, we've been using the [click this](/platform-redirect/?next=/directory/content/).

PlatformLink will use the context of your most recently viewed platform; for example, if you've been reading platform content, then been directed to a product page and then click back into platform content.

Reply:
So this link - [automatically attach trace headers to outgoing HTTP requests](/platforms/dotnet/performance/included-instrumentation/) - when viewed on the  ASP.NET page, sends you to the primary .NET page, which looks correct to me (that page is common to the .NET family).

If we want users to stay in the guide content, we'd want the link to use [automatically attach trace headers to outgoing HTTP requests](/platforms/dotnet/guides/aspnet/performance/included-instrumentation/).

Either way, the page content is the same.